docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG UBUNTU_VERSION=18.04
+FROM ubuntu:${UBUNTU_VERSION}
+
+# Use bash as the default shell
+RUN ln -snf /bin/bash /bin/sh
```

Comment:
Dockerfiles now support a `SHELL` command, which allows setting the default shell directly [1].  This technique is preferred as it uses the operating system specific command for setting the shell.  See the github issue [2] for the related discussion.

Example:

```
SHELL ["/bin/bash", "-c"]
RUN echo I am now using bash!
```

[1] https://docs.docker.com/engine/reference/builder/#shell
[2] https://github.com/moby/moby/issues/7281